test/Confluent.Kafka.IntegrationTests/Tests/Tests.cs

```diff
@@ -14,21 +14,73 @@
 //
 // Refer to LICENSE for more information.
 
+using Confluent.Kafka.Admin;
 using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Reflection;
 using Newtonsoft.Json.Linq;
-
+using Xunit;
 
 namespace Confluent.Kafka.IntegrationTests
 {
-    public static partial class Tests
+    public class GlobalFixture : IDisposable
+    {
+        private string bootstrapServers;
+
+        public GlobalFixture()
+        {
+            var assemblyPath = typeof(Tests).GetTypeInfo().Assembly.Location;
+            var assemblyDirectory = Path.GetDirectoryName(assemblyPath);
+            var jsonPath = Path.Combine(assemblyDirectory, "testconf.json");
+            var json = JObject.Parse(File.ReadAllText(jsonPath));
+            bootstrapServers = json["bootstrapServers"].ToString();
```

Comment:
Is there an automatic Json-to-type reader in .NET that you could use to deserialize JSON into a Configuration object right away?

Reply:
don't know - this is only one line, so i don't thinks warrants thinking about.